presto-product-tests/src/main/java/io/prestosql/tests/cassandra/TestInsertIntoCassandraTable.java

```diff
@@ -57,7 +58,7 @@ public Requirement getRequirements(Configuration configuration)
         return mutableTable(CASSANDRA_ALL_TYPES, CASSANDRA_INSERT_TABLE, CREATED);
     }
 
-    @Test(groups = CASSANDRA)
+    @Test(groups = {CASSANDRA, PROFILE_SPECIFIC_TESTS})
```

Comment:
why they are marked as `PROFILE_SPECIFIC_TESTS`?

Reply:
because they are specific to specific environment.

for example (before my changes):
```
$ ( cd presto-product-tests/src/main/java/io/prestosql/tests/sqlserver; git grep PROFILE_SPECIFIC_TESTS ; )
TestInsert.java:import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
TestInsert.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestInsert.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestInsert.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestInvalidSelect.java:import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
TestInvalidSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestInvalidSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestInvalidSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestSelect.java:import static io.prestosql.tests.TestGroups.PROFILE_SPECIFIC_TESTS;
TestSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
TestSelect.java:    @Test(groups = {SQL_SERVER, PROFILE_SPECIFIC_TESTS})
```